basketball_reference_web_scraper/data.py

```diff
@@ -43,13 +43,42 @@ class Team(Enum):
     UTAH_JAZZ = "UTAH JAZZ"
     WASHINGTON_WIZARDS = "WASHINGTON WIZARDS"
 
-    # DEPRECATED TEAMS
+    # RENAMED/DEFUNCT TEAMS
     CHARLOTTE_BOBCATS = "CHARLOTTE BOBCATS"
     NEW_JERSEY_NETS = "NEW JERSEY NETS"
     NEW_ORLEANS_HORNETS = "NEW ORLEANS HORNETS"
     NEW_ORLEANS_OKLAHOMA_CITY_HORNETS = "NEW ORLEANS/OKLAHOMA CITY HORNETS"
     SEATTLE_SUPERSONICS = "SEATTLE SUPERSONICS"
     VANCOUVER_GRIZZLIES = "VANCOUVER GRIZZLIES"
+    PHILADELPHIA_WARRIORS = "PHILADELPHIA WARRIORS"
+    SAN_FRANSISCO_WARRIORS = "SAN FRANSISCO WARRIORS"
+    BALTIMORE_BULLETS = "BALTIMORE BULLETS"
+    MINNEAPOLIS_LAKERS = "MINNEAPOLIS LAKERS"
+    ROCHESTER_ROYALS = "ROCHESTER ROYALS"
+    SYRACUSE_NATIONALS = "SYRACUSE NATIONALS"
+    ST_LOUIS_HAWKS = "ST. LOUIS HAWKS"
+    WASHINGTON_BULLETS = "WASHINGTON BULLETS"
+    MILWAUKEE_HAWKS = "MILWAUKEE HAWKS"
+    TRI_CITIES_BLACKHAWKS = "TRI CITIES BLACKHAWKS"
```

Comment:
I _think_ this should be `TRI-CITIES`? There might be couple other team names that are formatted in a specific way in the various HTML tables.

Reply:
Yes you're right. I'll make sure to fix that and double check the others